cognite/client/_api/assets.py

```diff
@@ -867,6 +880,10 @@ def filter(
                 ...     sort=(SortableAssetProperty.external_id, "asc"))
 
         """
+        warnings.warn(
+            "This method is deprecated. Use the list method with advanced_filter parameter instead.",
```

Comment:
I would add which method this is + a note on removal in the next major version of the SDK

Reply:
Updated